packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -121,6 +121,14 @@ export default class Run extends SfdxCommand {
       this.ux.warn(messages.getMessage('warningMessage'));
     }
 
+    // W-9346875 - default to human-readable result format for --wait flag
+    if (
+      this.flags.hasOwnProperty('wait') &&
+      !this.flags.hasOwnProperty('resultformat')
+    ) {
+      this.flags.resultformat = 'human';
+    }
+
```

Comment:
Just a heads up, I ran the code through the linter.  The conditional checks are still within one `if` statement, but the linter separated both checks on a single line into lines.